src/OXRS_MQTT.cpp

```diff
@@ -60,6 +61,9 @@ char * OXRS_MQTT::getTelemetryTopic(char topic[])
 void OXRS_MQTT::onConfig(callback callback)
 { 
   _onConfig = callback; 
+  char topic[64];

```

Comment:
Yeah I think this needs to be moved out of here and moved so it is updated after calls to `setClientId()`, `setTopicPrefix()` or `setTopicSuffix()`? This will ensure if the topic is ever changed, we refresh the screen.

I was thinking about this, we might want to display `+/osm-ad347f` as the topic - since we are actually publishing on `stat/` and `tele/` and listening on `conf/` and `/cmnd`. 

If you use `_getTopic(topic, "+")` you will get the topic with any prefix/suffix, ready for display.

Reply:
inserted screen updates as suggested.
Need to check how longer topics are displayed.